src/server/game/DungeonFinding/LFGMgr.cpp

```diff
@@ -2491,6 +2517,27 @@ namespace lfg
         GroupsStore[guid].SetDungeon(dungeon);
     }
 
+    void LFGMgr::SendMessageToAllPlayers(GuidUnorderedSet players, LfgProposal const& proposal)
+    {
+        // Send a message to the players
+        for (GuidUnorderedSet::const_iterator it = players.begin(); it != players.end(); ++it)
+        {
+            if (Player* player = ObjectAccessor::FindPlayer(*it))
+            {
+                ChatHandler handler = ChatHandler(player->GetSession());
+                LFGDungeonEntry const* dungeon = sLFGDungeonStore.LookupEntry(proposal.dungeonId);
+                if (dungeon)
+                {
+                    handler.PSendSysMessage(handler.GetAcoreString(LANG_LFG_TELEPORT_DUNGEON), dungeon->Name[0]);
+                }
+                else
+                {
+                    handler.PSendSysMessage(handler.GetAcoreString(LANG_LFG_TELEPORT_NO_DUNGEON));
+                }
```

Comment:
Now that it's a single line after if and else please remove the braces according to AC cpp code style.